evalml/pipelines/components/estimators/classifiers/catboost_classifier.py

```diff
@@ -42,6 +42,7 @@ def __init__(
         silent=True,
         allow_writing_files=False,
         random_seed=0,
+        n_jobs=-1,
```

Comment:
Might be time to finally write a docstring for catboost and xgboost? 😅  Especially since there's a risk users can pass in `n_threads` to Catboost and assume it will work.